keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1777,6 +1815,17 @@ type DeviceRequestAllocationResult struct {
     //
     // +required
     Device string
+
+	// AdminAccess is a copy of the AdminAccess value in the
```

Comment:
nit, I think your formatting is a bit off here.

Reply:
Will fix. Copy-and-paste from code adds tabs, while the README uses spaces.